src/Illuminate/Support/Str.php

```diff
@@ -99,10 +99,20 @@ public static function camel($value)
      *
      * @param  string  $haystack
      * @param  string|array  $needles
+     * @param  bool  $caseSensitive
      * @return bool
      */
-    public static function contains($haystack, $needles)
+    public static function contains($haystack, $needles, $caseSensitive = true)
     {
+        // Normalise case if the check is case insensitive
+        if ($caseSensitive === false) {
+            $haystack = strtolower($haystack);
+
+            $needles = collect($needles)->map(function ($item) {
```

Comment:
Does collect work on strings?

Reply:
That is, if needles is a single string?

Reply:
Yep, the collect function uses `getArrayableItems()` on the `Collection` class. The default return if none of the conditions are met is a `return (array) $items;` which creates an array with the single string.